src/test/resources/agreement.xml

```diff
@@ -0,0 +1,657 @@
+<document xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://netgrif.github.io/petriflow/petriflow.schema.xsd">
```

Comment:
A smallest reproducible example should be used for the unit test. Also the unit test nets should be generic and should not contain any business logic.